2-collectors/scala-stream-collector/src/test/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorServiceSpec.scala

```diff
@@ -286,5 +286,13 @@ class CollectorServiceSpec extends Specification {
           `Access-Control-Allow-Origin`(HttpOriginRange.`*`)
       }
     }
+
+    "cookieRedirectMacro" in {
+      "the collector should support a cookie replacement macro on redirect" in {
+        event.networkUserId = "1234"
+        val (res, Nil) = service.buildRedirectHttpResponse(event, "k", Map("u" -> "http://localhost/?uid=${SP_UUID}"))
+        res shouldEqual HttpResponse(302).withHeaders(`Location`("http://localhost/?uid=1234"))
+      }
```

Comment:
could you move this test to https://github.com/snowplow/snowplow/pull/3491/files#diff-88f19e201a23cbae2294497905a6b8a1R201 , you can make a local copy of the event if there are issues with setting the `networkUserId`